packages/flutter_tools/lib/src/localizations/gen_l10n_types.dart

```diff
@@ -149,6 +150,19 @@ ${List<String>.filled(4 + fileName.length + messageId.length + charNumber, ' ').
   final int charNumber;
 }
 
```

Comment:
Can you add a comment or two about what this is and where it is used?

Reply:
Fixed!